src/org/labkey/test/pages/core/login/DatabaseAuthenticationProvider.java

```diff
@@ -0,0 +1,31 @@
+package org.labkey.test.pages.core.login;
```

Comment:
This isn't a page. It should go in `org.labkey.test.params.login`, next to the base AuthenticationProvider.